### PR TITLE
azurem_signalr - check serviceMode is not empty

### PR DESCRIPTION
**Pull request by catriona-m**

### CODE REVIEW

File: internal/services/signalr/signalr_service_resource.go [hunk 1]
```diff
@@ -286,7 +286,10 @@ func resourceArmSignalRServiceCreate(d *pluginsdk.ResourceData, meta interface{}
 	} else {
 		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsEnableConnectivityLogs, strconv.FormatBool(connectivityLogsEnabled)))
 		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsEnableMessagingLogs, strconv.FormatBool(messagingLogsEnabled)))
-		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsServiceMode, serviceMode))
+
+		if serviceMode != "" {
```
Comment (tombuildsstuff) [inline internal/services/signalr/signalr_service_resource.go:290]: shouldn't this be defaulting this if it's not specified?

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
Left a question inline but this otherwise seems ok?

File: internal/services/signalr/signalr_service_resource.go (hunk 1 above)
Comment (catriona-m) [reply internal/services/signalr/signalr_service_resource.go:290]: This is computed in the schema at the minute until the old features block can be removed in 3.0 so I can't set a default there yet, but I could change it to be this if it makes more sense:

```
if serviceMode == "" {
  serviceMode!= "Default" 
}
```

Comment (tombuildsstuff) [reply internal/services/signalr/signalr_service_resource.go:290]: that'd work 👍

**Pull request merged**